examples/quarkus-jib-extension/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,94 @@
+####
```

Comment:
@pjgg I'm looking to test this example out. Could you provide some info into the significance of these two Dockerfiles?